packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -278,7 +278,7 @@ const ProductDetail = ({category, product, isLoading}) => {
                             />
                         }
                         recommender={'pdp-similar-items'}
-                        products={product && [product.id]}
+                        products={product && [{id: product.master.masterId, sku: product.id}]}
```

Comment:
Does `master` alwasy available in `product`?

Reply:
Currently yes since we only support master and variant products. This will need to change if we introduce other product types into the system.

I'll try to future proof this part

Reply:
@vcua-mobify Thank you! I'm already using standard products so future proofing this component will allow me to consume it without changes. I imagine some customers may also be.